File: Actuarialvaluations/clientapp/src/models/common/Dropdown.ts
/**
 * The Dropdown model class
 */
export class Dropdown {
    /**
     * The Value for dropdown
     */
    public value: number | null = null;

    /**
     * The Text Dropdown
     */
    public text: string | null = null;


    constructor(value: number, text: string) {
        if (text) {
            this.value = value;
            this.text = text;
        }
    }
}

File: Actuarialvaluations.Models/API/DropDown.cs
namespace Actuarialvaluations.Models.API
{
  /// <summary>
  /// The Markov state class
  /// </summary>
  public class DropDown : Entity
  {
    /// <summary>
    /// Gets or sets ID
    /// </summary>    
    public int Value { get; set; }

    /// <summary>
    /// Gets or sets Value
    /// </summary>
    public string Text { get; set; }
  }
}
